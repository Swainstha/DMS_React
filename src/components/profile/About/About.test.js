import testSnapFunction from '../../../util/testSnapFunction';
import About from './index'
import React from 'react';


testSnapFunction("<About />", "Snapshot test for About ",<About /> )