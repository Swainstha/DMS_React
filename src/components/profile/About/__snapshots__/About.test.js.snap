// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> Snapshot test for About  1`] = `ShallowWrapper {}`;
